controllers/databasecluster_controller.go

```diff
@@ -1444,12 +1495,15 @@ func (r *DatabaseClusterReconciler) reconcilePXC(ctx context.Context, req ctrl.R
 			}
 		}
 
-		if monitoring.Spec.Type == everestv1alpha1.PMMMonitoringType {
+		if monitoring.Spec.Type == everestv1alpha1.PMMMonitoringType { //nolint:nestif
 			image := defaultPMMClientImage
 			if monitoring.Spec.PMM.Image != "" {
 				image = monitoring.Spec.PMM.Image
 			}
 
+			//nolint:godox
+			// TODO (EVEREST-824): Set PMM container LivenessProbes and ReadinessProbes timeouts once possible.
```

Comment:
Why isn't it possible?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/blob/c066c6ef6e440dd2a523d718c8f1851de866d3f3/pkg/apis/pxc/v1/pxc_types.go#L514

It doesn't expose an API for it unlike the rest :/ 
I think we would have to create a ticket for it so that it gets added..

Reply:
Can you discuss this with Slava? If he agrees then please proceed to open a ticket for the appropriate projects in Jira [PXCO](https://perconadev.atlassian.net/browse/K8SPXC), [PSMDBO](https://perconadev.atlassian.net/browse/K8SPSMDB), [PGO](https://perconadev.atlassian.net/browse/K8SPG), [PSO](https://perconadev.atlassian.net/browse/K8SPS). Maybe Slava will tell you to open only for one of them :shrug: please confirm with him :slightly_smiling_face:

Reply:
If you decide to open a ticket then please update this comment with the ticket ID

Reply:
Done